integration-tests/jail/ssh/authorized_keys

```diff
@@ -0,0 +1,3 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC9VL9tO86mtzyI+Js3EzxGXGrH7uiYJyI8M4laBpmuRuxeS3aistg+LoQ9bOnDx4B6eVrXmr7BQ1zXZhtOSyJfiCvTV+3RZta89Rp9K1MuF6dcZA9nAtlsxW8vEnh0BEFtRlsrvJkR6LKrB3airMmTxjRb6SnLi0vD1DO0W4Y3n7hDhqdVruRBWt/oPDLPyFdu4Z9Ffk4uWPJqOgEk63qU+x2iiwgnIlHi7/NX9YQMXBKWCsJu5PVgyAAYfk86Wlq1l5xDQnBYK+V+tf6zMmlkLgBqJFXQca5UTqsSYPnW5AlxI6JIrJAOsxX+4e7Qf1sSil6hz1RHcKa+dtThEzqp  user@hongkong
```

Comment:
why is ssh needed for the test?

Reply:
this is is for debugging, i'll remove it

Reply:
it's debugging, i'll remove it